index.html

```diff
@@ -20,6 +20,14 @@ <h1 class="header_container_title"><span id="header">Real </span>Estate </h1>
           <li class="list_item_active"><a href="./index.html">Home</a></li>
           <li><a href="./about.html">About</a></li>
           <li><a href="#">Contact Us</a></li>
+
```

Comment:
I've tried to see your work in the browser, but only the HTML structure appears without CSS, 
that's because you add the stylesheet in the wrong way: 

```HTML
        <link rel="stylesheet" href="/style.css">
```

the right way 

```HTML
        <link rel="stylesheet" href="./style.css">
```